### ISSUE TITLE
Port test_baseprocess.py to Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8546]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8546|
|Type|enhancement|
|Created|2016-07-01 07:23:28Z|
|Branch|https://github.com/twisted/twisted/tree/8546-rodrigc-baseprocess|

twisted/internet/test/test_baseprocess.py needs to be ported to Python 3

<details><summary>Searchable metadata</summary>

```
trac-id__8546 8546
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8546_rodrigc_baseprocess 8546-rodrigc-baseprocess
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1467357808358213 1467357808358213
changetime__1467363733673308 1467363733673308
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/283

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_3"></a>|@Lukasa set owner to @rodrigc|
|-|-|

This is the best kind of porting: the kind that involves changing no code. LGTM, merge when ready.

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub set status to `closed`|
|-|-|

In changeset 5f9c663b503125c152e434cfb95a45feceab1d2b
```
#!CommitTicketReference repository="" revision="5f9c663b503125c152e434cfb95a45feceab1d2b"
Merge 8546-rodrigc-baseprocess: Run twisted.internet.test.test_baseprocess  on Python 3

Author: rodrigc
Reviewer: Lukasa
Fixes: #8546
```
